python/awaazde/utils.py

```diff
@@ -32,6 +32,7 @@ class CSVUtils(object):
     def write_or_append_to_csv(data, file_path, file_name, append=False):
         if data:
             keys = list(data[0].keys())
+            keys.extend(["error"])
```

Comment:
What happens if there is already an error in 1st record itself. In that case, there is already an error in keys. So when you add it again, Will there 2 error columns?

Reply:
Yes, there will be two Error columns. I missed this scenario. Updating code.